web/pipelines/steps-web-run-e2e-tests.yaml

```diff
@@ -8,3 +8,11 @@ steps:
     inputs:
       archiveFilePatterns: '${{ parameters.workspaceDir }}/e2e-tests/e2e-tests.zip'
       destinationFolder: '${{ parameters.workspaceDir }}/e2e-tests-files'
+  - task: qetza.replacetokens.replacetokens-task.replacetokens@3
```

Comment:
Please use latest version of this task `qetza.replacetokens.replacetokens-task.replacetokens@6`.

Examples can be found in `steps-run-terraform.yaml`

Reply:
done

Reply:
Please see the examples - the inputs are different between v3 and v6

Reply:
updated it accordingly.